adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -1276,6 +1294,45 @@
             return promise.promise;
         }
 
+        function check3pCookiesSupported() {
+            var promise = createPromise();
+
+            if (loginIframe.enable || kc.silentCheckSsoRedirectUri) {
+                var iframe = document.createElement('iframe');
+                iframe.setAttribute('src', kc.endpoints.thirdPartyCookiesIframe());
+                iframe.setAttribute('title', 'keycloak-3p-check-iframe' );
+                iframe.style.display = 'none';
+                document.body.appendChild(iframe);
+
+                var messageCallback = function(event) {
+                    if (iframe.contentWindow !== event.source) {
+                        return;
+                    }
+
+                    if (event.data !== "supported" && event.data !== "unsupported") {
```

Comment:
This breaks with **any** browser extension (e.g. FoxyGestures or React Dev Tools in Firefox) hat sends messages to new windows. Your code needs to be resilient to *any* message it does not recognize and not consider it an error.

Reply:
@ThiefMaster Thank you for reporting this. Could you please create a [JIRA](https://issues.redhat.com/browse/KEYCLOAK) for this (if you haven't already) so we can triage this effectively?

Reply:
@vmuzikar thanks for the comment. I opened a ticket on your JIRA [here](https://issues.redhat.com/browse/KEYCLOAK-16074).

Reply:
Just based on the code the fix looks like it will do the job :) If @saibot94 deploys it on his test instance (or if you have a test instance that's publicly accessible) I don't mind testing it to confirm it :)

Reply:
works fine!